### DISCUSSION TITLE
Skywalking connects to the ES database, and there is time behind the index to cause a mismatch

### DISCUSSION THREAD

**Discussion by joenzx** [apache/skywalking#8053] [General]
I have checked the parameter selection of ES7 in application.yaml, but there is no index format setting, why is this problem, which parameter is wrong
![d7ebc491e4ce785efec41b78882610a](https://user-images.githubusercontent.com/42016493/139817935-140b08df-f433-4ee2-b3ea-c0f804246c69.png)

![158d3039349f387357b02d602d892c0](https://user-images.githubusercontent.com/42016493/139818333-3f4e15f4-1c72-4be4-b455-b625de531d6d.png)
.

**Comment by wu-sheng** [comment]
It is very hard to follow your question, you didn't show clearly about your context.

For the missing part, it is not an index, it is an ElasticSearch template.

**Reply by joenzx** [reply]
> It is very hard to follow your question, you didn't show clearly about your context.
> 
> For the missing part, it is not an index, it is an ElasticSearch template.

This is my configuration of ES, other parameters are default
![image](https://user-images.githubusercontent.com/42016493/139835023-76b4b410-e7f3-4bff-80c8-208bba5e4e5d.png)

The error is ： "reason":"no such index [sw_metrics-histogram]"
Then I looked up all the indexes of ES and found this index sw_metrics-histogram-20211102

So you're saying that these two are not the same thing, and the correct context is both of them?

**Reply by wu-sheng** [reply]
Basically, we shouldn't debug for you, this is not how open source community works.

You could refer how we install SkyWalking on the k8s(KinD) through e2e process, 
https://github.com/apache/skywalking/blob/7aa0ba70483b621d40452b0b2f53423f59f039e5/test/e2e-v2/cases/istio/als/e2e.yaml?_pjax=%23js-repo-pjax-container%3Afirst-of-type%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%3Afirst-of-type%2C%20%5Bdata-pjax-container%5D%3Afirst-of-type#L46-L68
